### PR TITLE
attention augmented block should use BatchNorm

### PR DESCRIPTION
**Pull request by noahtren**
From the [paper](https://arxiv.org/pdf/1904.09925.pdf):
> In all our experiments, the augmented convolution is followed by a batch normalization [20] layer which can learn to scale the contribution of the convolution feature maps and the attention feature maps

This commit adds a Keras BatchNormalization layer to the output of an augmented conv2d block.

### CODE REVIEW

**Comment by noahtren** [comment]
I also want to thank you for this great implementation! Working great with TensorFlow 2 so far.

**Comment by titu1994** [comment]
Missed that part. Good catch !
